tagstudio/src/qt/resource_manager.py

```diff
@@ -31,6 +31,20 @@ def __init__(self) -> None:
                 )
             ResourceManager._initialized = True
 
+    @staticmethod
```

Comment:
why not a `classmethod` This code breaks if you subclass it.

Reply:
Similar explanation here: https://github.com/TagStudioDev/TagStudio/pull/390#discussion_r1739560040